packages/web/src/components/gcds-file-uploader/gcds-file-uploader.tsx

```diff
@@ -286,6 +280,32 @@ export class GcdsFileUploader {
     }
   }
 
+  /*
+   * Form internal functions
+   */
+  formResetCallback() {
+    this.internals.setFormValue('');
+    this.value = [];
+  }
+
+  formStateRestoreCallback(state) {
+    this.internals.setFormValue(state);
+    this.value = state;
+  }
+
+  /*
+   * Set form data for internals
+   */
+  private addFilesToFormData = files => {
+    const formData = new FormData();
+
+    for (let i = 0; i < files.length; i++) {
+      formData.append(this.name, files[i], files[i].name);
+    }
```

Comment:
If you use `forEach` here instead of `for`, the code will be a bit easier to read.
Something like this:
```
Array.from(files).forEach(file => {
    formData.append(this.name, file, file.name);
});
  ```